src/gatsby-node.js

```diff
@@ -29,6 +30,7 @@ export const sourceNodes = async (gatsbyContext, pluginOptions) => {
   createTypes(`
     type ${FEATURE_TYPE} implements Node {
       geometry: JSON!
+      raw: JSON!
```

Comment:
`raw` doesn't appear to be provided to the node in `createNode`?

Reply:
You're right, that's probably left over I forgot to delete.